index.html

```diff
@@ -6,6 +6,30 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>[cCc] Team Wilder Guides</title>
 
+    <!-- Adelle font from Adobe -->
+    <script type="text/javascript" src="//use.typekit.net/kmf0xzy.js"></script>
+    <script type="text/javascript">
+      try {
+        Typekit.load({ async: true });
+      } catch (e) {}
+    </script>
+
+    <!-- Rubik font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Rubik:wght@300&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- ludicrous font -->
```

Comment:
We might have wanted this with `src/css` but the other fonts import here - so I think this makes sense.

Reply:
Wasn't sure if it should be in
- reset.css
- its own css file in the css folder (then also included in index.html)
- tucked in the index like this

So yeah I put it next to all the other font imports so they're all in the same place at least.